### PR TITLE
[policies] Add CBL-Mariner policy

### PR DESCRIPTION
**Pull request by eric-desrochers**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X ] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/policies/distros/mariner.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright (C) Red Hat, Inc. 2019
```
Comment (pmoravec) [inline sos/policies/distros/mariner.py:1]: I guess this is worth amending after copy&paste ;-)

**Comment by pmoravec** [comment]
Apart of the three Flake syntax issues - what would be the purpose of sole policy? Until you add some Plugin class for it, it makes no sense..?

**Comment by eric-desrochers** [comment]
> Apart of the three Flake syntax issues - what would be the purpose of sole policy? Until you add some Plugin class for it, it makes no sense..?

The intent for now is to have a customize message instead of the the generic one when sos report is run.

**Comment by pmoravec** [comment]
ACK to the intent / policy itself. As I checked we do have several similar ones already.

Please fix the Flake errors and I guess also the Copyright comment.

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]
Ack. My only question would be is there an upload target MSFT customers/support would want to use for this policy?

**Comment by eric-desrochers** [comment]
> Ack. My only question would be is there an upload target MSFT customers/support would want to use for this policy?

Not yet.

**Pull request merged**